src/org/usfirst/frc/team467/robot/Autonomous/Actions.java

```diff
@@ -538,15 +502,21 @@ public static ActionGroup leftAdvancedSwitchRightScaleLeft() {
 		// lift elevator to place cube into scale
 		mode.addAction(elevatorToLowScale());
 		mode.addAction(releaseCube());
-		
+
 		return mode;
 	}
-	
+
+	public static ActionGroup testAction() {
```

Comment:
We really need to do a better job at naming these and adding Java doc comments describing what they do. The name testAction is about as generic and unuseful as it could be.

Reply:
The description is in the ActionGroupText.